index.js

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable */
```

Comment:
- [ ] hi team, great job implementing the mobile menu, however It is not recommended to disable eslint for the whole file, then there is no point in adding linter checks, I cannot yet approve the project, I suggest you to not disable eslint and rather try to fix those issues as they help you write more maintainable code.
What you can do is use the `npx eslint . --fix` command for autofixes and for other issue just use the click event in the index.js file itself rather than adding onClick as an inline attribute in the HTML file.

- [ ] The other thing is why do you have package.json and package-lock.json and linters json file in the gitignore :point_down: , please remove these files from your gitignore as they are needed for anybody else to clone and use this project with the required dependencies 
![image](https://user-images.githubusercontent.com/97521557/223059131-c344cba0-567c-449e-bf24-e547493be39b.png)